src/controllers/subscriptions.ts

```diff
@@ -260,16 +261,12 @@ export async function restartSubscription(req, res) {
 
 async function getRawSubs(opts = {}) {
   const options: { [k: string]: any } = { order: [['id', 'asc']], ...opts }
-  try {
-    const subs = await models.Subscription.findAll(options)
-    return subs
-  } catch (e) {
-    throw e
-  }
+  const subs = await models.Subscription.findAll(options)
+  return subs
```

Comment:
Should we be removing the try catch here?

Reply:
try catch with `throw e` in the catch block is useless. when an error is thrown in the try block, js will rethrow it in the catch block, so no difference except that with the useless catch block eslint starts to complain